.generate.yml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
seems like autogenerated, we need to fill in reasonable stuff here :)

Reply:
I changed it